frontend/src/app/application-dashboard/components/csi-info/csi-info.component.ts

```diff
@@ -29,24 +30,27 @@ export class CsiInfoComponent implements OnChanges {
       // Case 2: No CSI configuration for application
       this.errorCase = 2;
```

Comment:
if you use a string as errorCase you do not need comments that will deprecate over time